### ISSUE TITLE
captcha.website is blocked

### ISSUE THREAD

**Issue by Kippersz** [privacypass/challenge-bypass-extension#319]
"captcha.website is blocked
This page has been blocked by an extension
Try disabling your extensions."

Problem still persists even if the Privacy Pass extension is the only extension that is running. Google Chrome is on Version 101.0.4951.54 (latest).

Anybody else having this issue?

**Comment by Kippersz** [comment]
![image](https://user-images.githubusercontent.com/10134737/167249549-d95d7b97-2197-474e-945d-cf617f229daa.png)
This is the extension error message if it means anything.

**Comment by Chaoschaot232** [comment]
Same problem for me, can do waht I want with Version 101.0.4951.54 (Official Build) (64-bit) for chrome

**Comment by pradeep00950** [comment]
yes i also face same problem

**Comment by rhystedstone** [comment]
Here's the console's stack trace if it helps:
![image](https://user-images.githubusercontent.com/12123847/167531468-1e4226f4-a1bd-4956-bc8f-39f42a036087.png)
It seems to be on any site that uses Privacy Pass, but it's only occasional.

**Comment by armfazh** [comment]
CF experienced some issues [last week](https://www.cloudflarestatus.com/incidents/f6cv9cxtfdj2). Solved now.

**Comment by lordcyb3r** [comment]
still happening to me today. same error

**Comment by rhystedstone** [comment]
> CF experienced some issues [last week](https://www.cloudflarestatus.com/incidents/f6cv9cxtfdj2). Solved now.

This issue has been occuring for months, not a week.

**Comment by armfazh** [comment]
let's check some facts.
1) captcha.website is a site to create new tokens. The extension brings users to this page so user can get tokens.
2) Once the captcha is solved, 
2.1) extension register tokens that live in the extension. AND
2.2) redirects to the same page.

While redirection happens, the browser may show that "captcha page is blocked". This is sour but normal and immediately, the extension will get new tokens, and shows you _again_ the captcha website, presenting you the captcha solving interface.

The issue pointed few weeks ago was very specific and what people reported in the screnshots is accurate. This issue was already solved.

@rhystedstone you may have been observed a different issue, with similar or different symptoms tough.

**Comment by rhystedstone** [comment]
> let's check some facts.
> 1) captcha.website is a site to create new tokens. The extension brings users to this page so user can get tokens.
> 2) Once the captcha is solved, 
> 2.1) extension register tokens that live in the extension. AND
> 2.2) redirects to the same page.
> 
> While redirection happens, the browser may show that "captcha page is blocked". This is sour but normal and immediately, the extension will get new tokens, and shows you _again_ the captcha website, presenting you the captcha solving interface.
> 
> The issue pointed few weeks ago was very specific and what people reported in the screnshots is accurate. This issue was already solved.
> 
> @rhystedstone you may have been observed a different issue, with similar or different symptoms tough.
> 
> 

If it occurs again, I'll post a separate issue 🙂.

**Comment by bwims** [comment]
Still a problem with 3.0.5 

I have zero captcha points after seeing this:-
```
captcha.website is blocked
This page has been blocked by an extension
Try disabling your extensions.
ERR_BLOCKED_BY_CLIENT
```
